src/controller/model/yml/yml.go

```diff
@@ -0,0 +1,98 @@
+package yml
+
+import (
+    "strings"
+)
+
+const (
+    RestartAlways         = "always"
+    RestartNo             = "no"
+    FixedPortMapping      = "fix"
+    RandomPortMapping     = "random"
+    CustomizedPortMapping = "customized"
+)
+
+type Yml struct {
+    Infra    *Infra              `yaml:"infrastructure"`
+    Services map[string]*Service `yaml:"services"`
+    Runtime  *Runtime            `yaml:"runtime"`
+}
+
+type Infra struct {
+    Network []string `yaml:"network"`
+}
+
+type Service struct {
+    Image       string   `yaml:"image"`
+    Singleton   bool     `yaml:"singleton"`
+    Ports       []string `yaml:"ports"`
+    Config      *Config  `yaml:"config_file"`
+    Networks    []string `yaml:"networks"`
+    Volumes     []string `yaml:"volumes"`
+    Depends     []string `yaml:"depends_on"`
+    Environment []string `yaml:"environment"`
+}
+
+type Config struct {
+    Path    string `yaml:"path" json:"path"`
+    Mode    string `yaml:"mode" json:"mode"`
+    Content string `yaml:"content" json:"content"`
+}
+
+type Runtime struct {
+    Environment   []string                  `yaml:"env" json:"env"`
+    GlobalPolicy  *GlobalPolicy             `yaml:"global_policy" json:"global_policy"`
+    ServicePolicy map[string]*ServicePolicy `yaml:"services" json:"services"`
+}
+
+type GlobalPolicy struct {
+    Restart     string `yaml:"restart" json:"restart"`
+    PortMapping string `yaml:"port_mapping" json:"port_mapping"`
+}
+
+type ServicePolicy struct {
+    Restart     string `yaml:"restart" json:"restart"`
+    InstanceNum int    `yaml:"instance_num" json:"instance_num"`
+    PortMapping string `yaml:"port_mapping" json:"port_mapping"`
```

Comment:
I'm not so clear about your questions, generally speaking, all setting in "runtime" can be overrode while you click "create deployment", usually it gave deploy policies to our deploy service.

Instance:EntryPoint describes an instance's port mapping status, not policy, for example:
Instance redis has two entry points [(0.0.0.0:6379:6379/tcp), (0.0.0.0:80:80/tcp)], 6379 is redis connection port and 80 as a management port